docs/1-essentials/03-database.md

```diff
@@ -304,34 +304,41 @@ final class User
 
 The encryption key is taken from the `SIGNING_KEY` environment variable.
 
-### DTO properties
+### Data transfer object properties
 
-Sometimes, you might want to store data objects as-is in a table, without there needing to be a relation to another table. To do so, it's enough to add a serializer and caster to the data object's class, and Tempest will know that these objects aren't meant to be treated as database models. Next, you can store the object's data as a json field on the table (see [migrations](#migrations) for more info).
+It's possible to store objects as-is in a `json` column, when they don't need to be part of your database schema.
+
+To do this, you may simply mark these objects with the `#[Tempest\Mapper\SerializeAs]` and specify a name for this object to be serialized as. Not that this name must uniquely correspond to a single class.
 
 ```php
-use Tempest\Database\IsDatabaseModel;
-use Tempest\Mapper\CastWith;
-use Tempest\Mapper\SerializeWith;
-use Tempest\Mapper\Casters\DtoCaster;
-use Tempest\Mapper\Serializers\DtoSerializer;
+use Tempest\Mapper\SerializeAs;
 
-final class DebugItem
+final class User implements Authenticatable
 {
-    use IsDatabaseModel;
-    
-    /* … */
-    
-    public Backtrace $backtrace,
+    public PrimaryKey $id;
+
+    public function __construct(
+        public string $email,
+        #[Hashed, SensitiveParameter]
+        public ?string $password,
+        public Settings $settings,
+    ) {}
 }
 
-#[CastWith(DtoCaster::class)]
-#[SerializeWith(DtoSerializer::class)]
-final class Backtrace
+#[SerializeAs('user_settings')]
```

Comment:
I don't really understand what `user_settings` is referring to? Is it the database column name? Don't we already have `#[MapTo]` that we could maybe build on further? 

I understand the need for this feature, but the current naming is confusing to me